src/main/fc/rc.c

```diff
@@ -489,141 +492,176 @@ static FAST_CODE void processRcSmoothingFilter(void)
+                    // but I think this won't materially improve the extrapolation
+                    setpointSpeed = prevSetpointSpeed[axis] + prevSetpointSpeedDelta[axis];
+                    // pretend that there was stick movement also, to hold the same jitter value
+                    rcCommandDeltaAbs = prevRcCommandDeltaAbs[axis];
+                }
+            } else {
+                // for second and all subsequent duplicates...
+                // leave setpoint speed at zero
+                // zero the acceleration by setting previous speed to zero
+                // feedforward will smoothly decay and be attenuated by the jitter reduction value for zero rcCommandDelta
+                prevSetpointSpeed[axis] = 0.0f; // zero acceleration later on
+            }
+            prevPacketWasADuplicate[axis] = true;
         }
-        prevDuplicatePacket[axis] = true;
-    }
-    prevRxInterval = rxInterval;
+        prevRxInterval = rxInterval;
+    } else {
+        // don't interpolate for CRSF, ELRS, and maybe other systems that don't send duplicate packets on telemetry sends
+        setpointSpeed = setpointDelta * rxRate;
+    }
+
+    // calculate jitterAttenuation factor
+    // The intent is to attenuate feedforward when absolute rcCommandDelta is small, ie when sticks move very slowly
+    // Greater feedforward_jitter_factor values widen the attenuation range, and increase the suppression at center
+    // Stick input is the average of the previous two absolute rcCommandDelta values
+    // Output is jitterAttenuator, a value 0-1.0 that is a simple multiplier of the final feedforward value
+    // For the CLI user setting of feedforward_jitter_factor:
+    // User setting of 0 returns feedforwardJitterFactorInv = 1.0 (and disables the function)
+    // User setting of 1 returns feedforwardJitterFactorInv = 0.5
+    // User setting of 9 returns feedforwardJitterFactorInv = 0.1
+    // rcCommandDelta has 500 unit values either side of center stick position
+    // For a 250Hz link, a one second stick sweep center->max returns rcCommandDelta around 2.0
+    // For a user jitter reduction setting of 2, the jitterAttenuator value ranges linearly
+    // from 0.33 when rcCommandDelta is close to zero, up to 1.0 for rcCommandDelta of 2.0 or more
+    // For a user jitter reduction setting of 9, the jitterAttenuator value ranges linearly
+    // from 0.1 when rcCommandDelta is close to zero, up to 1.0 for rcCommandDelta is 9.0 or more
+    // note that the jitter reduction multiplies the final smoothed value of feedforward
+    // allowing residual smooth feedforward offsets even if the sticks are not moving
+    // this is an improvement on the previous version which 'chopped' FF to zero when sticks stopped moving
+    float jitterAttenuator = ((rcCommandDeltaAbs + prevRcCommandDeltaAbs[axis]) * 0.5f + 1.0f) * pid->feedforwardJitterFactorInv;
```

Comment:
(it may be better to make user setting independent of RC rate. `rxRate * rcCommandDeltaAbs` may work (but with value of ~500 for situation described above. It is done this way for feedforwardBoost )

Reply:
Yes, both the amount of feedforwardSmoothing, and the 'spread' of the effect of the jitter reduction, are influenced by rcRate.
It would be easier for users if there was automatic compensation.
However we have a lot of Presets that apply specific jitter and feedforwardSmoothing values depending on the rcRate and the flying style.  Most of these would 'break' if we did include an automatic compensation method.
Perhaps that can be done in a separate PR.